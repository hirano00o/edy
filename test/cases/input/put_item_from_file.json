{
  "ID": 14,
  "Name": "Mallory",
  "Email": "mallory@example.com",
  "Age": 20,
  "Birthday": {
    "Year": 2001,
    "Month": 8,
    "Day": 31
  }
}