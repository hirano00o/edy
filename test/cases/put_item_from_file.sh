#!/bin/bash

SCRIPT_ROOT_DIR=$1
TEST_NAME=$(basename "$0" | sed "s/\..*//")

# aws dynamodb put-item --table-name User \
#   --item file://${SCRIPT_ROOT_DIR}/cases/input/put_item_from_file.json \
#   --endpoint-url http://localhost:8000
CMD="edy p -t User -I ${SCRIPT_ROOT_DIR}/cases/input/put_item_from_file.json --local 8000"

. "${SCRIPT_ROOT_DIR}"/helper.sh

FILTER_CONDITION="ID,N = 14"

run_such_put_helper
