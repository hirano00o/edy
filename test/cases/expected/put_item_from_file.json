[
  {
    "Age": 20,
    "Birthday": {
      "Day": 31,
      "Month": 8,
      "Year": 2001
    },
    "Email": "mallory@example.com",
    "ID": 14,
    "Name": "Mallory"
  }
]
